README.md

```diff
@@ -168,6 +168,10 @@ export OC_CLUSTER_PASS=<password of the cluster user>
                              (Optional)
                              By default - false
 
+    --report-suffix        - Add suffix to the tests basename.
```

Comment:
Please, move this argument under "Tests arguments:".

And add it here as well - https://github.com/stolostron/acmqe-mcn-test/blob/main/lib/common/helper_functions.sh#L106